FlowCryptCommon/FlowcryptTimer.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  FlowcryptTimer.swift
+//  FlowCryptCommon
+//
+//  Created by Ioan Moldovan on 4/6/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import CoreFoundation
+
+class FlowcryptTimer {
+
+    private var startTime: CFAbsoluteTime?
+    private var endTime: CFAbsoluteTime?
+
+    func start() {
+        startTime = CFAbsoluteTimeGetCurrent()
+    }
+
+    func stop() {
+        endTime = CFAbsoluteTimeGetCurrent()
+    }
+
+    var duration: CFAbsoluteTime {
```

Comment:
Let's add units for clarity. Physics values should always come with units.

Let's make this `durationMs`

Reply:
ok